### PR TITLE
Makes README easier to read.

### PR DESCRIPTION
**Pull request by harrysarson**
Hopefully none of these changes effect the meaning of the text but make the document easier to read.

Would love to hear what you think about these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,17 +55,17 @@ See also:
...
-and will (asynchronously) `cb(null, data)` once for each call.
+and will (asynchronously) call `cb(null, data)` once for each call.
 
 To signify an end state, the stream eventually returns `cb(err)` or `cb(true)`.
-When indicating a terminal state, `data` *must* be ignored.
+When signifying an end state, `data` *must* be ignored.
 
 The `read` function *must not* be called until the previous call has called back.
-Unless, it is a call to abort the stream (`read(truthy, cb)`).
+Unless, it is a call to abort the stream due to an error (`read(truthy, cb)`).
```
Comment (regular) [inline README.md]: this edit changes the meaning. It can actually abort the stream in absence of an error.

**Comment by dominictarr** [comment]
thanks @harrysarson merged into 3.6.2

File: README.md (hunk 1 above)
Comment (harrysarson) [reply README.md]: ah I see, so `read` can be called early if 
1) there is an error (`read(Error, cb)`)
2) no error but the sink is no longer interested in values (`read(true, cb)`)

Is that right?

Comment (harrysarson) [reply README.md]: @dominictarr could you clarify this? I can then fix this and we can merge

**Comment by dominictarr** [comment]
that is correct, you can call read again before the previous callback
if and only if the new call is an abort (err or true), in which case
the previous callback will be called with the abort end|error, then
the abort callback will be called with the same end|error

**Comment by harrysarson** [comment]
@regular how does bd30193 look?

**Comment by harrysarson** [comment]
@dominictarr @regular

Can this be merged?

**Comment by dominictarr** [comment]
merged into 3.6.8

**Comment by dominictarr** [comment]
@harrysarson thanks for the good work, I would like to invite you to pull-stream org! then you can just merge things yourself (it's a good practice to still use PRs though, so that others have a chance to comment)

**Comment by harrysarson** [comment]
Thanks Dominic :)

**Pull request merged**